Resources/Prototypes/Recipes/Lathes/medical.yml

```diff
@@ -224,3 +224,12 @@
   materials:
     Steel: 100
     Wood: 200
+
+- type: latheRecipe
+  id: MedTek
+  result: MedTekCartridge
+  completetime: 8
+  materials:
+    Glass: 600
+    Gold: 400
+    Silver: 200
```

Comment:
I am pretty sure you can use the `BaseCartridgeRecipe` here as well, even though it is defined in `cargo.yaml`.